awx/main/signals.py

```diff
@@ -363,6 +364,18 @@ def model_serializer_mapping():
     }
 
 
+def emit_activity_stream_change(instance):
```

Comment:
Any time we trigger a signal for Activity Stream create/update/delete/associate, just schedule a post-commit call that notifies the external logger.

Reply:
This has the nice side effect of *actually making this work* outside of HTTP requests (like if you call an `awx-manage` command).